ci-wheel.Dockerfile

```
@@ -25,84 +25,88 @@ ENV PIP_EXTRA_INDEX_URL="https://pypi.anaconda.org/rapidsai-wheels-nightly/simpl
+      debianutils build-essential software-properties-common \
+      jq wget gcc zlib1g-dev libbz2-dev \
+      libssl-dev libreadline-dev libsqlite3-dev libffi-dev curl git libncurses5-dev \
+      libnuma-dev openssh-client libcudnn8-dev zip libopenblas-dev liblapack-dev \
+      protobuf-compiler autoconf automake libtool cmake yasm libopenslide-dev
+    add-apt-repository ppa:git-core/ppa
+    add-apt-repository ppa:ubuntu-toolchain-r/test
+    apt update -y
+    apt install -y git gcc-9 g++-9
+    add-apt-repository -r ppa:git-core/ppa
+    add-apt-repository -r ppa:ubuntu-toolchain-r/test
+    update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 90 --slave /usr/bin/g++ g++ /usr/bin/g++-9 --slave /usr/bin/gcov gcov /usr/bin/gcov-9
+    rm -rf /var/lib/apt/lists/*
+    ;;
+  "centos"*)
+    yum update --exclude=libnccl* -y
+    yum install -y epel-release
+    yum update --exclude=libnccl* -y
+    yum install -y \
+      which wget gcc zlib-devel bzip2 bzip2-devel readline-devel sqlite \
+      sqlite-devel xz xz-devel libffi-devel curl git ncurses-devel numactl \
+      numactl-devel openssh-clients libcudnn8-devel zip blas-devel lapack-devel \
+      protobuf-compiler autoconf automake libtool centos-release-scl scl-utils cmake \
+      yasm openslide-devel
+    yum remove -y git
+    yum install -y https://packages.endpointdev.com/rhel/7/os/x86_64/endpoint-repo.x86_64.rpm
+    yum install -y git jq devtoolset-11
+    yum remove -y endpoint-repo
+    yum clean all
+    echo -e ' \
+      #!/bin/bash\n \
+      source scl_source enable devtoolset-11\n \
+    ' > /etc/profile.d/enable_devtools.sh
+    pushd tmp
+    wget https://ftp.openssl.org/source/openssl-1.1.1k.tar.gz
+    tar -xzvf openssl-1.1.1k.tar.gz
+    cd openssl-1.1.1k
+    ./config --prefix=/usr --openssldir=/etc/ssl --libdir=lib no-shared zlib-dynamic
+    make
+    make install
+    popd
+    ;;
+  "rockylinux"*)
+    dnf update -y
+    dnf install -y epel-release
+    dnf update -y
+    dnf install -y \
+      which wget gcc zlib-devel bzip2 bzip2-devel readline-devel sqlite \
+      sqlite-devel xz xz-devel libffi-devel curl git ncurses-devel numactl \
+      numactl-devel openssh-clients libcudnn8-devel zip jq openslide-devel \
+      protobuf-compiler autoconf automake libtool dnf-plugins-core cmake
+    dnf config-manager --set-enabled powertools
+    dnf install -y blas-devel lapack-devel
+    dnf -y install gcc-toolset-11-gcc gcc-toolset-11-gcc-c++
+    dnf -y install yasm
+    dnf clean all
+    echo -e ' \
+      #!/bin/bash\n \
+      source /opt/rh/gcc-toolset-11/enable \
+    ' > /etc/profile.d/enable_devtools.sh
+    pushd tmp
+    wget https://ftp.openssl.org/source/openssl-1.1.1k.tar.gz
+        tar -xzvf openssl-1.1.1k.tar.gz
+    cd openssl-1.1.1k
+    ./config --prefix=/usr --openssldir=/etc/ssl --libdir=lib no-shared zlib-dynamic
+    make
+    make install
+    popd
+    ;;
+  *) 
+    echo "Unsupported LINUX_VER: ${LINUX_VER}" && exit 1
```

```suggestion
    echo "Unsupported LINUX_VER: ${LINUX_VER}"
    exit 1
```

Let's remove the `&&` since we're no longer using it anywhere else.